src/cli/generate/bundle/Bundle.definition.ts

```diff
@@ -23,11 +23,26 @@ import { PortOption } from "./options/Port.option";
 export const BundleDefinition: ICommandDefinition = {
     name: "bundle",
     aliases: ["b", "bun", "bund"],
-    summary: "Generates a Bundle",
-    description: "CICS Bundle meta-data is generated within the working directory. " +
+    summary: "Generates a bundle",
+    description: "Generate CICS bundle within the working directory. " +
```

Comment:
Wording is slightly different to whats below
```suggestion
    description: "Generate a CICS bundle in the working directory. " +
```